webapp/channels/src/components/tabs/tabs/index.tsx

```diff
@@ -35,6 +35,7 @@ export default function Tabs({
             onSelect={onSelect}
             className={classNames('tabs', className)}
             mountOnEnter={mountOnEnter}
+            autoFocus={true}
```

Comment:
This shouldn't actually be autofocused since this is a component we're going to want to use elsewhere. We generally shouldn't be changing the users' focus outside of if a new window or element opens. 

There's an article about that here:
https://www.boia.org/blog/accessibility-tips-be-cautious-when-using-autofocus